charts/corda/values.yaml

```
@@ -67,8 +67,8 @@ kafka:
   replicas: 3
   # -- prefix to use for Kafka topic names (to support the use of a single Kafka cluster by multiple Corda clusters)
   topicPrefix: ""
-  # -- Specifies whether we should delete the previous topics with the given prefix before trying to create new ones. Uses provided prefix, and NOPs if set to true, but no topics are found
-  cleanUp: false
+  # -- Specifies whether we should delete the previous topics with the given prefix before trying to create new ones. Uses provided prefix, or assumes no prefix if none provided, and NOPs if set to true, but no topics are found

```

Just to make it really explicit...

```suggestion
  # -- Specifies whether existing topics with the given prefix should be deleted before trying to create new ones (deletes all existing topics if no prefix is given)
```